impl/src/test/java/org/ehcache/GettingStarted.java

```diff
@@ -178,6 +178,29 @@ public void testPersistentDiskCache() {
 
     persistentCacheManager.close();
   }
+  
+  @Test
+  public void testPersistentDiskCacheWithDefaultPersistentMode() {
+    CacheConfiguration<Long, String> cacheConfiguration = CacheConfigurationBuilder.newCacheConfigurationBuilder()
+        .withResourcePools(ResourcePoolsBuilder.newResourcePoolsBuilder().heap(10, EntryUnit.ENTRIES).disk(100, EntryUnit.ENTRIES).build())
+        .buildConfig(Long.class, String.class);
+
+    PersistentCacheManager persistentCacheManager = CacheManagerBuilder.newCacheManagerBuilder()
+        .with(new PersistenceConfiguration(new File(System.getProperty("java.io.tmpdir") + "/persistent-cache-data")))
+        .withCache("persistent-cache", cacheConfiguration)
+        .build(true);
+
+    Cache<Long, String> cache = persistentCacheManager.getCache("persistent-cache", Long.class, String.class);
+
+    // Comment the following line on subsequent run and see the test pass
+    cache.put(42L, "That's the answer!");
+    assertThat(cache.get(42L), is("That's the answer!"));
```

Comment:
The assertion is comparing strings here. While is() actually works (it does ==), it's more correct to use equalTo() (which calls equals()).